environments/64-test/public.yml

```diff
@@ -0,0 +1,177 @@
+# For a real deployment
+# should be e.g. 'www.commcarehq.org'
+internal_domain_name: 'localdomain'
+
+etc_hosts_lines: []
+etc_hosts_lines_removed: []
+
+touchforms_enabled: false
+
+riak_backend: "bitcask"
+riak_ring_size: 64
+
+ansible_user_password_sha_512: "{{ secrets.ANSIBLE_USER_PASSWORD_SHA_512 }}"
+
+dev_users:
+  present:
+    - dmiller
+    - droberts
+    - jwilson
+    - npellegrino
+  absent: []
+
+# use this value to switch between nginx and apache
+proxy_type: nginx
+
+elasticsearch_endpoint: '{{ groups.elasticsearch.0 }}'
+elasticsearch_memory: '128m'
+elasticsearch_cluster_name: 'deves'
+
+# uncomment to only allow access to eth1
+#ufw_private_interface: eth1
+
+# uncomment and change to the control machine IP when using ufw
+# this ensures that ssh access from the control machine is always allowed regardless of other firewall rules
+#control_machine_ip: 127.0.0.1
+
+ECRYPTFS_PASSWORD: "{{ secrets.ECRYPTFS_PASSWORD }}"
+
+KSPLICE_ACTIVE: no
+KSPLICE_ACTIVATION_KEY: "{{ secrets.KSPLICE_ACTIVATION_KEY }}"
+
+DATADOG_ENABLED: True
+DATADOG_API_KEY: "{{ secrets.DATADOG_API_KEY }}"
+DATADOG_APP_KEY: "{{ secrets.DATADOG_API_KEY }}"
+
+CLOUDANT_CLUSTER_NAME: null
+
+
+# This dumps the dummy keystore, but allows the ansible script to continue, should be overridden for real deploys
+#keystore_file: 'DimagiKeyStore'
+
+backup_es: False
+
+postgres_users:
+  commcare:
+    username: commcarehq
+    password: commcarehq
+  devreadonly:
+    username: "devreadonly"
+    password: devreadonly
+    role_attr_flags: 'NOSUPERUSER,NOCREATEROLE,NOCREATEDB'
+    privs:
+      -
+        privs: SELECT
+        objs: ALL_IN_SCHEMA
+  replication:
+      username: 'hqrepl'
+      password: 'hqrepl'
+      role_attr_flags: 'LOGIN,REPLICATION'
+
+encrypted_root: '/opt/data'
+
+backup_postgres: plain
+backup_blobdb: True
+postgresql_backup_master: True
+postgres_s3: False
+nadir_hour: 18
+
+nofile_limit: 65536
+
+supervisor_http_enabled: True
+supervisor_http_username: "{{ secrets.SUPERVISOR_HTTP_USERNAME }}"
+supervisor_http_password: "{{ secrets.SUPERVISOR_HTTP_PASSWORD }}"
+
+mailrelay: 'relay.example.com'
+
+couchdb2:
+  username: "{{ localsettings_private.COUCH_USERNAME }}"
+  password: "{{ localsettings_private.COUCH_PASSWORD }}"
+
+couchdb_bind:
+  ty: static
+  host: '{{ hostvars[inventory_hostname]["ansible_eth0"]["ipv4"]["address"] }}'
+
+couch_dbs:
+  default:
+    host: "{{ groups.couchdb2.0 }}"
+    port: "{{ couchdb2_proxy_port }}"
+    name: commcarehq
+    username: "{{ localsettings_private.COUCH_USERNAME }}"
+    password: "{{ localsettings_private.COUCH_PASSWORD }}"
+    is_https: False
+  users:
```

Comment:
Don't think you need to specify the `users` DB separately, you can just leave it out.